packages/design-system-scripts/gulp/docs/generatePages/generateReactExample.js

```diff
@@ -0,0 +1,91 @@
+const MemoryFS = require('memory-fs');
+const path = require('path');
+const savePage = require('./savePage');
+const webpack = require('webpack');
+
+/**
+ * Creates an HTML page using the React component and no additional UI.
+ * This can then be viewed in a browser, or rendered in an iFrame in
+ * the documentation.
+ * @param {Object} page
+ * @param {String} docsPath
+ * @param {String} rootPath - Root docs site path
+ * @return {Promise}
+ */
+function generateReactExample(page, docsPath, rootPath) {
+  return new Promise((resolve, reject) => {
+    if (rootPath) {
+      rootPath = `${rootPath}/`;
+    }
+
+    const examplePath = path.resolve(__dirname, '../../../packages', page.reactExamplePath);
```

Comment:
This is an example of a line that will need to change. This is from the old script, when this script was in this relative location to the packages directory. It's not necessary to use `__dirname` at all anymore. The new version of this line might be as simple as:
```js
const examplePath = path.resolve(page.reactExamplePath);
```